src/components/ResourcesCard.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import InformationCards from "./cards/InformationCards";
+import HeaderCArd from "./cards/HeaderCArd";
+
+interface Props {
+  Entries: any;
+}
+
+const ResourcesCard = ({ Entries }: Props) => {
```

Comment:
These should be the direct props of this component `{ title, description, entries, colSpan, rowSpan, heightMultiplier } `.

Reply:
fixed

Reply:
fixed